src/metax_api/models/catalog_record.py

```diff
@@ -1195,6 +1199,13 @@ def _pre_create_operations(self, pid_type=None):
 
             self.date_cumulation_started = self.date_created
 
+        if (self.catalog_is_ida() or self.catalog_is_att()):
+            if 'issued' not in self.research_dataset:
+                try:
```

Comment:
I don't think try-except is necessary here since we are only using internal values and functions